test/e2e/google_compute.go

```diff
@@ -32,15 +33,22 @@ func createGCEStaticIP(name string) (string, error) {
 	// NAME           REGION      ADDRESS       STATUS
 	// test-static-ip us-central1 104.197.143.7 RESERVED
 
-	output, err := exec.Command("gcloud", "compute", "addresses", "create",
-		name, "--project", testContext.CloudConfig.ProjectID,
-		"--region", "us-central1", "-q").CombinedOutput()
-	if err != nil {
+	var output []byte
+	var err error
+	for retries := 0; retries < 5; retries++ {
+		output, err = exec.Command("gcloud", "compute", "addresses", "create",
+			name, "--project", testContext.CloudConfig.ProjectID,
+			"--region", "us-central1", "-q").CombinedOutput()
+		if err == nil {
+			break
+		}
 		glog.Errorf("Creating static IP with name:%s in project: %s", name, testContext.CloudConfig.ProjectID)
 		glog.Errorf("output: %s", output)
+		time.Sleep(2 * time.Second)
```

Comment:
nit:Similar comment to previous one.  Please use exponential back along the lines of "sleep $((retries * 5))" rather than an arbitrary static sleep period.

nit: Why 'retries' here and 'attempt' above.  Prefer to be consistent.